admin_interface/templatetags/admin_interface_tags.py

```diff
@@ -80,6 +80,13 @@ def get_admin_interface_setting(setting):
     return getattr(theme, setting)
 
 
+@simple_tag()
+def get_inline_template(template):
```

Comment:
Rename the template tag to `get_admin_interface_inline_headerless_template` to avoid possible clashes.

Reply:
Updated